patches/chrome-browser-ui-views-frame-multi_contents_view.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/ui/views/frame/multi_contents_view.cc b/chrome/browser/ui/views/frame/multi_contents_view.cc
+index 8490634465a47fbaa6477477e427d30857815930..e21996f0614c62929bf770dd9f8b33b776cbeb5b 100644
+--- a/chrome/browser/ui/views/frame/multi_contents_view.cc
++++ b/chrome/browser/ui/views/frame/multi_contents_view.cc
+@@ -55,7 +55,7 @@ MultiContentsView::MultiContentsView(
+ 
+   SetLayoutManager(std::make_unique<views::DelegatingLayoutManager>(this));
+   contents_container_views_.push_back(
+-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
++      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
```

Comment:
Should these be proper BRAVE_ override definitions + patches?

Reply:
cc: @cdesouza-chromium